supplier/views.py

```diff
@@ -0,0 +1,13 @@
+from django.shortcuts import render
+from supplier.models import Supplier
+
+
```

Comment:
In here you should have something like :
@login_required(login_url='/example url you want redirect/') #redirect when user is not logged in

Redirect him to a page that says cannot access this page